Content.Shared/Throwing/ThrowingSystem.cs

```diff
@@ -0,0 +1,106 @@
+using Content.Shared.Interaction;
+using Content.Shared.Tag;
+using Robust.Shared.Physics;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Throwing;
+
+public sealed class ThrowingSystem : EntitySystem
+{
+    public const float ThrowAngularImpulse = 1.5f;
+
+    /// <summary>
+    /// The minimum amount of time an entity needs to be thrown before the timer can be run.
+    /// Anything below this threshold never enters the air.
+    /// </summary>
+    public const float FlyTime = 0.15f;
+
+    /// <summary>
+    ///     Tries to throw the entity if it has a physics component, otherwise does nothing.
+    /// </summary>
+    /// <param name="entity">The entity being thrown.</param>
+    /// <param name="direction">A vector pointing from the entity to its destination.</param>
+    /// <param name="strength">How much the direction vector should be multiplied for velocity.</param>
+    /// <param name="user"></param>
+    /// <param name="pushbackRatio">The ratio of impulse applied to the thrower - defaults to 10 because otherwise it's not enough to properly recover from getting spaced</param>
+
+    [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
+    [Dependency] private readonly ThrownItemSystem _thrownSystem = default!;
+    [Dependency] private readonly TagSystem _tagSystem = default!;
+    
+    public void TryThrow(
```

Comment:
xmldocs should be on the actual function

Reply:
ah oops. Ill go fix that